chapter_linear-networks/linear-regression-scratch_vn.md

```diff
@@ -197,10 +198,10 @@ For example, it requires that we load all data in memory and that we perform lot
 The built-in iterators implemented in Apache MXNet are considerably efficient and they can deal both with data stored on file and data fed via a data stream.
```

Comment:
> Kích thước của các đặc trưng trong mỗi minibatch cho ta biết kích thước của batch lẫn kích thước của các đặc trưng đầu vào.

> The shape of the features in each minibatch tells us both the minibatch size and the number of input features.

the number of input features --> số lượng đặc trưng đầu vào thì đúng hơn. Kích thước hay liên tưởng tới chiều spatial.